resources/views/helper.php

```diff
@@ -11,16 +11,15 @@
     exit("This file should not be included, only analyzed by your IDE");
 }
 
-<?php foreach($namespaces as $namespace => $aliases): ?>
+<?php foreach($namespaces_by_extends_ns as $namespace => $aliases): ?>
 <?php if ($namespace == '\Illuminate\Database\Eloquent'): continue; endif; ?>
-namespace <?= $namespace == '__root' ? '' : trim($namespace, '\\') ?> {
+namespace <?= $namespace == '__root' ? '' : trim($namespace, '\\') ?> { 
 <?php foreach($aliases as $alias): ?>
 
     <?= $alias->getClassType() ?> <?= $alias->getExtendsClass() ?> {
         <?php foreach($alias->getMethods() as $method): ?>
 
-        <?= trim($method->getDocComment('        ')) ?>
-
+        <?= trim($method->getDocComment('        ')) ?> 
```

Comment:
Here is an extra whitespace after closing tag - `?> `. Minor but creates unnecessary diff.